nixos/modules/services/web-apps/lemmy.nix

```diff
@@ -63,174 +86,110 @@ in
             default = "medium";
             description = lib.mdDoc "The difficultly of the captcha to solve.";
           };
-        };
-
-        options.database.createLocally = mkEnableOption "creation of database on the instance";
 
+          options.database.createLocally = mkEnableOption "creation of database on the instance";
```

Comment:
This should not be contained inside `options.captcha` (it was not like this before).